roles/common/tasks/endpoint_tests.yml

```diff
@@ -6,9 +6,12 @@
   changed_when: false
   register: output
 
-- name: Check that endpoint exist {{ common_endpoint_test_id }}
+- name: |
+    TEST Check that {{ item[0] }} {{ item[2] }} endpoint exists
+    {{ common_endpoint_test_id }}
   ansible.builtin.assert:
     that:
       - output.stdout != ""
     success_msg: "The endpoint {{ item[0] }} is found."
-    fail_msg: "The endpoint for service {{ item[0] }} "
\ No newline at end of file
+    fail_msg: "The endpoint for service {{ item[0] }} "
+    fail_msg: "The endpoint for service {{ item[0] }} on the {{ item[2] }} interface was not found"
```

Comment:
I'm not sure if these 2 lines are intentional. Did you want to delete L16?

Reply:
Well, this is embarassing, this one is definitely not intentional